src/store/modules/aeternity.js

```diff
@@ -114,95 +97,125 @@ export default {
           },
         });
         commit('setSdk', { instance });
-        await dispatch('initTippingContractIfNeeded');
       }
     },
     async scanForWallets({ commit, state: { sdk } }) {
-      const scannerConnection = await BrowserWindowMessageConnection({
-        connectionInfo: { id: 'spy' },
-      });
-      const detector = await WalletDetector({ connection: scannerConnection });
+      let stopScan;
+      let resolve = null;
+      let rejected = (e) => {
+        throw e;
+      };
       // eslint-disable-next-line no-underscore-dangle
       const webWalletTimeout = this._vm.$isMobileDevice ? 0
         : setTimeout(() => commit('enableIframeWallet'), 10000);
 
-      return new Promise((resolve) => {
-        detector.scan(async ({ newWallet }) => {
-          if (!newWallet) return;
+      async function handleNewWallet({ newWallet }) {
+        try {
+          if (!newWallet) return null;
```

Comment:
this check seems to be not necessary
https://github.com/aeternity/aepp-sdk-js/blob/2743e4af828902d90e532447b6c526c833869e49/src/aepp-wallet-communication/wallet-detector.ts#L34-L50